README.md

```diff
@@ -1,2 +1,124 @@
-# doogie_robot
-Robot packages for Doogie Mouse
+# **doogie_robot**
+
+The **doogie_robot** stack provides a [ROS] interface communication to Doogie Mouse **hardware** plataform. For tutorials, please check [doogie_robot/Tutorials] on wiki.
+
+**Keywords:** Micromouse, Gazebo, ROS
+
+**Author: [Mateus Meneses]<br />
+Affiliation: [BIR - Brazilian Institute of Robotics]<br />
+Maintainer: Mateus Meneses, mateusmenezes95@gmail.com **
+
+### Supported Versions
+
+- **Kinetic**: Built and tested under [ROS] Kinetic and Ubuntu 16.04
+
+<!-- [![Build Status](http://rsl-ci.ethz.ch/buildStatus/icon?job=ros_best_practices)](http://rsl-ci.ethz.ch/job/ros_best_practices/) TODO -->
+
+### Dependencies 
+- [doogie_base] : ROS packages stack with all common packages for working with Doogie Mouse
+
+<p align="center">
+   <img src="docs/doogie_robot.jpeg" alt="doogie_robot hw" title="Example Image">
+</p>
+</br>
+
+</br>
+
+# **Table of Contents**
+- [**doogie_robot**](#doogierobot)
+    - [Supported Versions](#supported-versions)
+    - [Dependencies](#dependencies)
+- [**Table of Contents**](#table-of-contents)
+- [**File System**](#file-system)
+- [**Installation**](#installation)
+    - [Building from Source:](#building-from-source)
+  - [Example of Usage](#example-of-usage)
+    - [Launch Doogie Mouse at Gazebo:](#launch-doogie-mouse-at-gazebo)
+- [**Purpose of the Project**](#purpose-of-the-project)
+- [**License**](#license)
+- [**Bugs & Feature Requests**](#bugs--feature-requests)
+
+# **File System**
+
+- [doogie_bringup] : Package with launch files to bringup Doogie Mouse robot controllers.
+- [doogie_drivers]: Cotains drivers used by Doogie Mouse robot **URDF**.
+
+</br>
+
+# **Installation**
+
+<!-- ### 1. Installation from Packages:
+
+TODO
+
+    sudo apt-get install ros-indigo-...
+
+
+or you could also build this repository from source. -->
+
+### Building from Source:
+
+Attention, if you haven't installed [ROS] yet, please check [Ubuntu install of ROS Kinetic](http://wiki.ros.org/kinetic/Installation/Ubuntu). Desktop-Full Install is the recommended one in order to work with this repository.    
+
+**Building:**
+
+First, lets create a catkin workspace.
+
```

Comment:
We can put a note here advertising whether the `doogie_ws` already exists, don't need to make again